### DISCUSSION TITLE
What do “Attach” and “Detach” specifically mean? I couldn’t find an explanation for these terms in the current documentation.

### DISCUSSION THREAD

**Discussion by w7team** [longhorn/longhorn#9713] [General]
Attach 和 Detach具体是什么意思，目前文档里没有找到相关解释，所以来这里请教一下：
1，我们目前测试机器有两个节点，创建完Volume后发现需要手动Attach才行，不明白这一步操作具体是干啥用的，能否实现自动Attach。
2，Attach时需要选择host，我一般需要选择哪个节点的host，如果我的pod在节点2，Attach时选择节点1会不会有影响？

What do “Attach” and “Detach” specifically mean? I couldn’t find an explanation for these terms in the current documentation, so I’m here to ask for clarification:
1,Currently, we have two nodes on our test machine. After creating the Volume, we found that we need to manually Attach it, and I’m not clear on what this step does exactly. Is it possible to automate the Attach process?
2,When attaching, we need to select a host. Which node’s host should I generally choose? If my pod is on node 2, will it have any impact if I choose the host from node 1 during the Attach process?

**Comment by derekbit** [comment]
Hello @w7team,
You can see the Kubernetes document https://kubernetes.io/docs/concepts/storage/.

**Comment by w7team** [comment]
![image](https://github.com/user-attachments/assets/6f6fdd8c-103f-4adf-ae33-d54c552a5e56)
我是说这个功能没有具体介绍

**Reply by derekbit** [reply]
Hello, as I mentioned, what you need is to understand the volume attachment and detachment that you can find in the Kubernetes document. In addition, https://longhorn.io/ is also helpful.

![image](https://github.com/user-attachments/assets/3ca5fc1e-f53d-44a8-bcac-47f5d64bd13f)

**Reply by w7team** [reply]
也就是说，如果在k8s内，我只需要在Volume创建pvc就自动Attach了，不需要再手动Attach host了对吧？因为我看到默认是detached状态，以为还需要手动Attach host

So, within k8s, once I create the PVC for the Volume, it will automatically Attach, and I don’t need to manually Attach the host, right? Because I saw that the default state is detached, and I thought I would still need to manually Attach the host.

**Reply by akashsemil** [reply]
Yes, If you create a PVC it will be automatically attached to the workload, Longhorn UI do provide flexibility to create a volume and attach it to specific host manually.
